support-frontend/assets/pages/[countryGroupId]/router.tsx

```diff
@@ -0,0 +1,25 @@
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import { renderPage } from 'helpers/rendering/render';
+import { geoIds } from 'pages/geoIdConfig';
+import { Checkout } from './checkout';
+
+const router = createBrowserRouter(
+	geoIds
+		.map((geoId) => [
```

Comment:
could you use a `flatMap` here instead of a `map` and then a `flat` later on?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap

Reply:
Indeed I can, and have!